src/dataLoaders/components/collectorsWizard/select/StreamingSelector2.tsx

```diff
@@ -21,8 +21,9 @@ import {
 } from 'src/writeData/constants/contentTelegrafPlugins'
 
 // Types
+import {TelegrafPlugin as TelegrafPlugin2} from 'src/types/dataLoaders'
```

Comment:
```suggestion
import {TelegrafPlugin as DataLoaderTelegrafPlugin} from 'src/types/dataLoaders'
```

`TelegrafPlugin2` is a little bit too vague I think

Reply:
good catch! not a good name indeed